custom/conf/app.example.ini

```
@@ -1578,6 +1578,10 @@ LEVEL = Info
 ;; Prefix displayed before subject in mail
 ;SUBJECT_PREFIX =
 ;;
+;; Set the Return-Path header, to let mail clients reply to a different mail as it was send from.
+;; If '<>' is set, it indicates has the same efect as if you would set a noreply address for FROM.
```

```suggestion
;; Set the Return-Path header where bounced e-mails will be sent to.
```

The second sentence seems superflous, I don't understand the relation to noreply addresses.